packages/blockchain-wallet-v4-frontend/src/modals/Staking/AccountSummary/AccountSummary.template.success.tsx

```diff
@@ -0,0 +1,383 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { Exchange } from '@core'
+import { formatFiat } from '@core/exchange/utils'
+import {
+  CoinType,
+  EarnAccountBalanceResponseType,
+  EarnEligibleType,
+  FiatType,
+  StakingLimitsType,
+  StakingRatesType
+} from '@core/types'
+import { Button, Icon, Link, Text } from 'blockchain-info-components'
+import CoinDisplay from 'components/Display/CoinDisplay'
+import FiatDisplay from 'components/Display/FiatDisplay'
+import { EarnStepMetaData } from 'data/types'
+
+import { OwnProps } from '.'
+import Detail from './AccountSummary.detail.template'
+import {
+  Bottom,
+  Container,
+  DetailsItemContainer,
+  DetailsWrapper,
+  LineVectorDetails,
+  LinkWrapper,
+  Row,
+  StatusIconWrapper,
+  StatusSupplyWrapper,
+  StatusWrapper,
+  Top,
+  TopText,
+  WarningContainer,
+  Wrapper
+} from './AccountSummary.model'
+
+const AccountSummary: React.FC<Props> = (props) => {
+  const {
+    accountBalances,
+    coin,
+    flagEDDInterestFileUpload,
+    handleClose,
+    handleDepositClick,
+    handleUpLoadDocumentation,
+    handleWithdrawalSupplyInformation,
+    showSupply,
+    stakingEligible,
+    stakingRates,
+    stepMetadata,
+    walletCurrency
+  } = props
+
+  const { coinfig } = window.coins[coin]
+  const account = accountBalances && accountBalances[coin]
+  const currencySymbol = Exchange.getSymbol(walletCurrency) as string
+
+  const accountBalanceBase = account && account.balance
+  const stakingBalanceBase = account && account.totalRewards
+
+  const isDepositEnabled = stakingEligible[coin] ? stakingEligible[coin]?.eligible : false
+  const { commission, rate } = stakingRates[coin]
+
+  return (
+    <Wrapper>
```

Comment:
You can get rid of a lot of this flyout structure code by importing FlyoutContainer, FlyoutHeader, FlyoutContent, FlyoutFooter components.